--- diagram.lua	2024-11-18 23:55:02.673989033 +0100
+++ diagram.patched.lua	2024-11-19 00:10:14.359878550 +0100
@@ -158,7 +158,7 @@
         write_file(infile, code)
         pipe(
           self.execpath or 'mmdc',
-          {"--pdfFit", "--input", infile, "--output", outfile},
+          {"--pdfFit", "--puppeteerConfigFile", "/usr/local/share/puppeteer/config.json", "--input", infile, "--output", outfile},
           ''
         )
         return read_file(outfile), mime_type
